lib/ansible/modules/cloud/vmware/vcenter_extension_info.py

```diff
@@ -31,17 +32,17 @@
 '''
 
 EXAMPLES = r'''
-- name: Gather facts about vCenter Extensions
-  vcenter_extension_facts:
+- name: Gather info about vCenter Extensions
+  vcenter_extension_info:
     hostname: '{{ vcenter_hostname }}'
     username: '{{ vcenter_username }}'
     password: '{{ vcenter_password }}'
-  register: ext_facts
+  register: ext_info
   delegate_to: localhost
 '''
 
 RETURN = r'''
-extension_facts:
+extension_info:
```

Comment:
You must not change/rename module return values. This will break people's playbooks!

Reply:
fixed